qtpydocking/dock_container_widget.py

```diff
@@ -227,8 +228,14 @@ def drop_into_container(self,
         insert_param = dock_area_insert_parameters(area)
         floating_dock_container = floating_widget.dock_container()
 
-        new_dock_areas = floating_dock_container.findChildren(
-            DockAreaWidget, '', Qt.FindChildrenRecursively)
+        api = os.environ['QT_API']
```

Comment:
I think the preferred check here may be to use `qtpy.API`, which tells you exactly which one qtpy chose after its logic (see [here](https://github.com/spyder-ide/qtpy/blob/275fe456dc7095162dbd2054a1d866bc28eefacc/qtpy/__init__.py#L216)).

Reply:
Fixed. Also, rather than using `qtpy.API`, I used the specific `boolean` variables they provided (much more specific)